Functions/xASL_adm_Dicom2Parms.m

```diff
@@ -93,7 +93,8 @@
 	% this should not give an error, but multiple ASL scale slopes should give an error
 	% Check if FLAIR occurs
 	isASL = true; % by default, assume these are ASL DICOMs
-	if ~isempty(findstr(FileList{1},'FLAIR')) || ~isempty(findstr(FileList{1},'T1'))
```

Comment:
This edit could be replaced by ~isempty(regexpi(FileList{1}, '(FLAIR|T1|T2|T1c)'))

Reply:
The whole thing was useless as isASL parameter is never used :) Deleted completely.

Reply:
This was probably used in the past